public/app/features/dashboard-scene/scene/DashboardSceneUrlSync.ts

```diff
@@ -116,3 +118,41 @@ export class DashboardSceneUrlSync implements SceneObjectUrlSyncHandler {
     }
   }
 }
+
+interface ResolveInspectPanelByKeyState extends SceneObjectState {
+  panelKey: string;
+}
+
+class ResolveInspectPanelByKey extends SceneObjectBase<ResolveInspectPanelByKeyState> {
+  constructor(state: ResolveInspectPanelByKeyState) {
+    super(state);
```

Comment:
what is the value of this being a standalone behavior vs being just part of PanelInspectDrawer ?  

it does not look like it can be reused.

Reply:
It can't but if we decide to move the `PanelInspectDrawer` to scenes lib we have one dependency on DashboardScene less.